dask/dataframe/io/tests/test_sql.py

```diff
@@ -519,3 +519,11 @@ def test_to_sql_kwargs():
             TypeError, match="to_sql\\(\\) got an unexpected keyword argument 'unknown'"
         ):
             ddf.to_sql("test", uri, unknown=None)
+
+
+def test_to_sql_engine_kwargs(caplog):
```

Comment:
I'm wondering if this test should be more extensive like `test_extra_connection_engine_keywords` and also do a `read_sql_table` to ensure the result is the same.

Reply:
Thanks! I added a few more scenarios that could be affected by the new code but weren't covered before, namely `echo=False` and `assert read(write(X)) == X`.